templates/intro-services/README.md

```diff
@@ -68,31 +70,38 @@ Use the following command to deploy your app as `my_service`.
 !serve deploy main:my_app --name=my_service
 ```
 
-**Tip**: if your app has PyPI dependencies added from the workspace, `serve deploy` will automatically compile these dependencies into a Docker image prior to deploying to optimize startup time.
+**Tip**: If your app has PyPI dependencies that you added from the workspace, `serve deploy` automatically compiles these dependencies into a Docker image prior to deploying to optimize startup time.
 
-### Service UI Overview
+### Service Overview page in the console
 
-Navigate to your newly created service in the Anyscale UI (`Home > Services > my_service`). It should be in "Starting" state. Click into it and wait for the service to enter "Active" state.
+Navigate to your newly created service in the Anyscale console at **Home > Services > my_service**. It should be in the "Starting" state. Click the service name and wait for the service to enter the "Running" state.
 
-You should see the service state, key metrics, and system event logs on the overview page.
+You should see the service state, key metrics, and system event logs on the Overview page.
 
 <img src="https://raw.githubusercontent.com/anyscale/templates/main/templates/intro-services/assets/service-overview.png" height=400px>
 
-### Query from the public Internet
+### Query from the public internet
 
-Once the service is up, you can query the service from the public Internet using the same logic as when testing it locally, with two changes:
+Once the service is running, query the service from the public internet using similar logic from the test query in the development workspace. Make two changes:
 1. Update the `HOST` to the service endpoint.
 2. Add the authorization token as a header in the HTTP request.
 
-Both of these values are printed when you run `serve deploy`. You can also find them on the service page. For example, if the output looks like:
+Find the `HOST` and authorization token values in one of two ways:
+- Run `serve deploy`
```

Comment:
```suggestion
- Look at the output of `serve deploy`
```

Reply:
telling ppl to re-run it to get the token is a little strange

Reply:
Good point. Will fix.